src/main.cc

```diff
@@ -51,7 +51,8 @@ void Serialization(std::ostream *stream,
     *stream << std::flush;
 }
 
-int ProcessAST(const Config& config, std::unique_ptr<std::istream>& in, std::unique_ptr<std::ostream>& out) {
```

Comment:
This file had them on new line except for these 2 functions, so I moved them for consistency, but it will be a moot point soon.